OpenTelemetry.sln

```diff
@@ -152,7 +152,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentati
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{7C87CAF9-79D7-4C26-9FFB-F3F1FB6911F1}"
 	ProjectSection(SolutionItems) = preProject
-		docs\getting-started.md = docs\getting-started.md
+		docs\logs\building-your-own-exporter.md = docs\logs\building-your-own-exporter.md
```

Comment:
Is there a simple way to manage this (instead of having to update this file each time we add/move/remove a markdown file)? I don't use Visual Studio at all and this for me is a pure manual work without any CI validation.

Reply:
Ugh yea, solution folders are kind of annoying in this respect. Not sure if there exists better support around this. One thought (maybe a bad one) that just popped into my mind is that we could make a `csproj` for the docs. A project, unlike a solution, has a relationship to an actual folder on disk. Might make this easier to manage without having to think about it.